mesop/web/src/editor/editor.ts

```diff
@@ -34,16 +34,23 @@ class Editor {
     // Hotkey for hot reload
     //
     // Binds:
-    // cmd + shift + r (MacOs)
+    // cmd + shift + r (MacOS)
+    // alt + shift + r (ChromeOS - to avoid conflict with browser's hard reload)
     // ctrl + shift + r (Other platforms)
-    if (
-      event.key === 'r' &&
-      (isMac() ? event.metaKey : event.ctrlKey) &&
-      event.shiftKey
-    ) {
+    if (event.key !== 'r' || !event.shiftKey) {
+      return;
+    }
+
+    const isMacOS = isMac();
+    const isChrome = isChromeOS();
+    const shouldReload = 
+      (isMacOS && event.metaKey) ||
+      (isChrome && event.altKey) ||
+      (!isMacOS && !isChrome && event.ctrlKey);
+
```

Comment:
I think it could be cleaner to separate the hot key logic to a "shouldReload()" function.

Right now it's not that easy to read. It's ok to have some redundancy if we can be slightly more explicit about the hot key mappings.

if (isMacOs) {
 return event.metaKey and event.shiftKey and event.key === 'r'
}
if isChrome {
  return ...
}
return ....